src/main/java/world/anhgelus/molehunt/Molehunt.java

```diff
@@ -60,25 +113,31 @@ public void onInitialize() {
         ).then(
                 literal("hide").executes(context -> {
                     timerVisibility.put(context.getSource().getPlayer(), false);
-                    context.getSource().sendFeedback(() -> Text.of("Hiding molehunt timer"), false);
+                    context.getSource().sendFeedback(() -> Text.translatable("commands.molehunt.timer.hide"), false);
                     return Command.SINGLE_SUCCESS;
                 })
         ));
-        command.then(literal("moles").requires(source -> (game != null) && game.isAMole(source.getPlayer())).executes(context -> {
-            context.getSource().sendFeedback(() -> Text.literal("List of moles: " + game.getMolesAsString()),false);
+        command.then(literal("moles").executes(context -> {
+            final var source = context.getSource();
+            if (game == null || !game.isAMole(source.getPlayer())) {
+                source.sendFeedback(() -> Text.translatable("commands.molehunt.moles.list.deny"),false);
+                return Command.SINGLE_SUCCESS;
+            }
+            source.sendFeedback(() -> Text.translatable("commands.molehunt.moles.list").append(" " + game.getMolesAsString()),false);
             return Command.SINGLE_SUCCESS;
         }));
         command.then(literal("stop").requires(source -> source.hasPermissionLevel(1)).executes(context -> {
             if (game == null || !game.hasStarted()) {
-                context.getSource().sendError(Text.of("Game has not started yet"));
-                return Command.SINGLE_SUCCESS;
+                throw (new SimpleCommandExceptionType(Text.translatable("commands.molehunt.stop.failed"))).create();
             }
 
             game.stop();
 
             return Command.SINGLE_SUCCESS;
         }));
 
+        ServerLifecycleEvents.SERVER_STARTED.register(server -> CONFIG = new Config(server));
```

Comment:
You could load the config file before, and only send packets on this event. I feel like it would be better, even though it probably won't make a real difference

Reply:
It's not a config file, it's a config linked with the gamerules. We can't do this before because the Minecraft Server is not ready

Reply:
oh yeah my bad